### PR TITLE
Implement Docket Switch Review/Confirm (Grant)

### PR DESCRIPTION
**Pull request by jcq**
Connects #15102

Jira: [CASEFLOW-297](https://vajira.max.gov/browse/CASEFLOW-297)

### Description
Please explain the changes you made here.

### Acceptance Criteria
- [x] Code compiles correctly
- [x] After a user selects "continue" on the Add/Remove tasks page, they are taken to a confirmation page
- [ ] The progress bar should be on the last step (Review and Confirm)
- [ ] Please format the header and instructions based on the designs below
- [ ] The first part of the instructions includes "You are switching from [Existing docket] to [New docket]. Tasks specific to the [Existing docket] docket will be automatically removed, and tasks associated with the [New docket] docket will be automatically created."
- [ ] Please add in the fields that are displayed according to the design (please note - these slightly differ for a full grant vs a partial grant - partial has a "Issues on original docket" section
- [ ] Final action is "Confirm docket switch"
- [ ] Selecting "back" takes the user to the Add/Remove tasks page
- [ ] Redirect to the Case Details page (see Granted Docket switch banner ticket)

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---

### Code Documentation Updates
- [ ] Add or update code comments at the top of the class, module, and/or component.

### Storybook Story
*For Frontend (Presentationa) Components*
* [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file (e.g. create `MyComponent.stories.js` alongside `MyComponent.jsx`)
* [ ] Give it a title that reflects the component's location within the overall Caseflow hierarchy
* [ ] Write a separate story (within the same file) for each discrete variation of the component

### CODE REVIEW

**Comment by jcq** [comment]
## Screenshots

### Full Grant, Veteran Claimant
Screen Shot 2021-02-03 at 16 55 31

### Partial Grant, Veteran Claimant
Screen Shot 2021-02-03 at 16 55 45

### Full Grant, Alternate Claimant
Screen Shot 2021-02-03 at 16 56 08

File: client/app/queue/docketSwitch/grant/DocketSwitchReviewConfirm.stories.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { sub } from 'date-fns';
```
Comment (Sjones352) [inline client/app/queue/docketSwitch/grant/DocketSwitchReviewConfirm.stories.js:2]: We have a DateUtil and was wondering does {sub } do?

Comment (jcq) [reply client/app/queue/docketSwitch/grant/DocketSwitchReviewConfirm.stories.js:2]: Subtracts a given amount of time (based on a specified period) from a given date

**Review by Sjones352 — APPROVED** [review]
This looks really good. @leikkisa can you please give one final review.
Screen Shot 2021-02-05 at 2 32 42 PM

**Review by leikkisa — APPROVED** [review]
An approval for posterity (after accidentally not approving it and another approval), because this looks great. Thanks for the extra fixes too!

File: app/models/serializers/work_queue/appeal_serializer.rb [hunk 2]
```diff
@@ -16,7 +16,8 @@ class WorkQueue::AppealSerializer
         notes: issue.notes,
         diagnostic_code: issue.contested_rating_issue_diagnostic_code,
         remand_reasons: issue.remand_reasons,
-        closed_status: issue.closed_status
+        closed_status: issue.closed_status,
```
Comment (wdrougas-tista) [inline app/models/serializers/work_queue/appeal_serializer.rb:19]: Is this serializing appeals data when an appeal is in a BVA employee's queue?

Comment (jcq) [reply app/models/serializers/work_queue/appeal_serializer.rb:19]: Yeah, this is for anytime (?) it's used during Queue

**Pull request merged**